bin/add_host_to_ssh_known_hosts

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -eu
+
+# Parameter can be just a host name, or a full http, https or git URL. Defaults to `github.com`.
```

Comment:
I'm not sure I love this as a default, IMHO it might just make sense to just fail if no URL is provided? 

As much as GitHub is widely used, making this tool GitHub-centric feels weird?

Reply:
Fixed in 51c9c8f